src/content/en/tools/chrome-user-experience-report/getting-started.md

```diff
@@ -226,18 +222,15 @@ towards the bins under 1000 ms, let’s compute the density sum for this range.
 ```sql
 #standardSQL
 SELECT
-  SUM((
-    SELECT
-      SUM(bin.density)
-    FROM
-      UNNEST(first_contentful_paint.histogram.bin) bin
-    WHERE
-      bin.start < 1000
-      AND origin = 'http://example.com')) AS density
+    SUM(bin.density) AS density
 FROM
-  `chrome-ux-report.all.201710`
+    `chrome-ux-report.chrome_ux_report.201710`,
```

Comment:
As an aside, could we create a 'latest' alias such that our queries don't fall out of date?